components/server/src/workspace/workspace-download-service.ts

```diff
@@ -49,11 +49,7 @@ export class WorkspaceDownloadService {
                     return;
                 }
 
-                const bucketName = this.storageClient.bucketName(userId);
-                const path = `/workspaces/${workspaceId}/full.tar`;
-                const signedUrl = await this.storageClient.createSignedUrl(bucketName, path, "read", {
-                    promptSaveAs: `${workspaceId}.tar`
-                });
+                const signedUrl = await this.storageClient.createWorkspaceContentDownloadUrl(userId, workspaceId);
```

Comment:
The `promptSaveAs: ${workspaceId}.tar` functionality didn't make it through this migration. There currently isn't a `storage` option that does this.

Reply:
Haven't found a way to do this with the go lib yet. I'll have another look.

Reply:
I created an issue for this in order to unblock this PR for now: https://github.com/gitpod-io/gitpod/issues/3438

Depending on how importing this is for us, we can decide whether we want to investigate this further in the near future or not.